src/DashboardService/Controllers/PrioritiesController.cs

```diff
@@ -0,0 +1,57 @@
+using LT.DigitalOffice.DashboardService.Business.Priority;
+using LT.DigitalOffice.DashboardService.Models.Dto.Models;
+using LT.DigitalOffice.DashboardService.Models.Dto.Requests.Priority;
+using LT.DigitalOffice.DashboardService.Models.Dto.Requests.Priority.Filters;
+using LT.DigitalOffice.Kernel.Responses;
+using Microsoft.AspNetCore.JsonPatch;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.DashboardService.Controllers;
+
+[Route("[controller]")]
+[ApiController]
+public class PrioritiesController : ControllerBase
+{
+  [HttpPost]
+  public async Task<OperationResultResponse<Guid?>> CreateAsync(
+    [FromBody] CreatePriorityRequest request,
+    [FromServices] CreatePriorityCommand command)
+  {
+    return await command.ExecuteAsync(request);
+  }
+  
+  [HttpGet]
+  public async Task<FindResultResponse<PriorityInfo>> GetAsync(
+    [FromServices] GetPrioritiesCommand command,
+    [FromQuery] GetPrioritiesFilter filter)
+  {
+    return await command.ExecuteAsync();
+  }
+  
+  [HttpGet("{id}")]
+  public async Task<OperationResultResponse<PriorityInfo>> GetAsync(
+    [FromRoute] Guid id,
+    [FromServices] GetPriorityCommand command)
+  {
+    return await command.ExecuteAsync(id);
+  }
+
+  [HttpPatch("{id}")]
+  public async Task<OperationResultResponse<bool>> PatchAsync(
+    [FromRoute] Guid id, 
+    [FromBody] JsonPatchDocument<PatchPriorityRequest> request,
+    [FromServices] EditPriorityCommand command)
+  {
+    return await command.ExecuteAsync(id, request);
+  }
+  
+  [HttpDelete("{id}")]
+  public async Task<OperationResultResponse<bool>> RemoveAsync(
+    [FromRoute] Guid id,
+    [FromServices] RemovePriorityCommand command)
```

Comment:
I do not understand in which rule you place  [FromRoute], [FromServices] and [FromBody]. At each endpoint they change their location

Reply:
++